src/data/routes.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "routeName": "MUNNAR-MARAYOOR-KANTHALLOOR-CHINNAR",
+        "routeId": "R1",
+        "routeColorName": "Voilet",
+        "routeColor": "#543585",
+        "routeDescription": "Giving Name Violet is for the presence of Neelakurinji plant in Eravikulam National park and Jacaranda in kanthanalloor Route. Both the flowers are violet in colour."
+    },
+    {
+        "routeName": "Munnar - Adimali",
+        "routeId": "R2",
+        "routeColorName": "Indigo",
+        "routeColor": "#41458C",
+        "routeDescription": "The highest tea plantation which is closest to the sky ( colour of the sky is blue ) is located in Kolukkumalai, one of the destinations in this route."
+    },
+    {
+        "routeName": "Munnar - Suryanelli - Kolukkumala - Chokramudi Peak",
+        "routeId": "R3",
+        "routeColor": "#3089AE",
+        "routeColorName": "Blue",
+        "routeDescription": "This Route is full of Grasslands, green in colour which is also a plastic free zone"
+    },
+    {
+        "routeName": "Munnar - Meesapulimala",
+        "routeId": "R4",
+        "routeColorName": "Green",
+        "routeColor": "#41AD63",
+        "routeDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        "routeIllustrationUrl": "https://image.freepik.com/free-vector/purple-starry-night-sky-landscape-with-pine-forest-illustration_105940-382.jpg" 
+    },
+    {
+        "routeName": "Munnar - CSI Church - Marayoor",
+        "routeId": "R5",
+        "routeColorName": "Yellow",
+        "routeColor": "#EDC237",
+        "routeDescription": "Heritage route is welcome for people who are more into history and heritages, it has destinations related to old stuffs. The colour yellow represents old.",
+        "routeIllustrationUrl": "https://image.freepik.com/free-vector/purple-starry-night-sky-landscape-with-pine-forest-illustration_105940-382.jpg" 
+    },
+    {
+        "routeName": "Munnar - Mankulam - Anakulam",
+        "routeId": "R6",
+        "routeColorName": "Orange",
+        "routeColor": "#E47E2E",
+        "routeDescription": "The presence of Orange Cultivation",
+        "routeIllustrationUrl": "https://image.freepik.com/free-vector/purple-starry-night-sky-landscape-with-pine-forest-illustration_105940-382.jpg" 
```

Comment:
@tomahawk-pilot  Upload this assets in s3 and provide URL for the team. We can't depend on public image servers